MockKit/MockCGMManager.swift

```diff
@@ -450,6 +450,8 @@ public final class MockCGMManager: TestingCGMManager {
 
     public let shouldSyncToRemoteService = false
     
+    public static let healthKitStorageDelay: TimeInterval = .minutes(5)
```

Comment:
Would be nice to have this configurable. Not a blocker.

Reply:
Since it is `static` you would have to delete and re-add the CGM simulator, but at least that's doable.